R/FilterPerformance.R

```diff
@@ -79,7 +79,8 @@ FilterPerformance = R6Class("FilterPerformance",
 
       perf = map_dbl(fn, function(x) {
         task$col_roles$feature = x
-        resample(task, self$learner, self$resampling, clone = character())$aggregate()
```

Comment:
Thanks for the fix, ideally one seperates issues into different branches

Reply:
yes yes